internal/plugins/scorecard/init.go

```diff
@@ -0,0 +1,259 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
IMO it should be with the other SDK customizations in `operator-sdk/internal/util/plugins/`

Reply:
I'm hesitant to start throwing things into `internal/util/plugins` like we did with the bundle stuff. Eventually we'll have a bunch of unrelated phase 2-like plugin code in there, and have the mess we currently have in a bunch of `internal/util` subpackages. The scorecard config scaffold will eventually be its own phase 2 plugin I expect, so should live here.

Reply:
I agree with @estroz's sentiment. Maybe as a follow-up we should move the "OLM"-like plugin stuff from `operator-sdk/internal/util/plugins/` back into  `internal/plugins/olm`

And then also try to refactor them to look as much like what we think phase 2 plugins will look like. Given that we're duplicating the `kubebuilder` machinery code, I think we could refactor these to use the scaffold and template interfaces.

Reply:
with you will do olm, scorecard and etc .. then I am ok with. I just do not like the idea of each requirement be solved in a diff way.